app/ts-cli/cmd/interactive.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2022 Huawei Cloud Computing Technologies Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(interactiveCmd)
+}
+
+var (
+	interactiveCmd = &cobra.Command{
+		Use:   "interactive",
+		Short: "Work in interactive mode",
```

Comment:
Make sure the first letter of the message is consistent in lower case or upper case.